docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -82,75 +82,12 @@ If successful, your robot will show that it's **Live**.
 
 For more `micro-rdk-installer` CLI usage options, see [GitHub](https://github.com/viamrobotics/micro-rdk/tree/main/micro-rdk-installer).
 
-### Configure an esp32 board
-
-{{< alert title="Info" color="info" >}}
-
-The`esp32` [board](/components/board/) model is not currently provided for you as a built-in option in [the Viam app](https://app.viam.com), so you cannot use the **Config Builder** to configure this board.
-
-{{< /alert >}}
-
-To add an `esp32` board, navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com) and select **Raw JSON** mode.
-
-Copy the following JSON template and paste it into your configuration inside the `"components"` array:
-
-{{< tabs name="Configure an esp32 Board" >}}
-{{% tab name="JSON Template"%}}
-
-```json {class="line-numbers linkable-line-numbers"}
-{
-  "attributes": {
-    "pins": [
-      <int>
-    ],
-    "analogs": [
-      {
-        "pin": "<number>",
-        "name": "<your-analog-name>"
-      }
-    ]
-  },
-  "depends_on": [],
-  "model": "esp32",
-  "name": "<your-board-name>",
-  "type": "board"
-}
-```
-
-{{% /tab %}}
-{{% tab name="JSON Example" %}}
-
-```json {class="line-numbers linkable-line-numbers"}
-{
-  "attributes": {
-    "pins": [15],
-    "analogs": [
-      {
-        "pin": "34",
-        "name": "sensor"
-      }
-    ]
-  },
-  "depends_on": [],
-  "model": "esp32",
-  "name": "board",
-  "type": "board"
-}
-```
-
-{{% /tab %}}
-{{< /tabs >}}
-
-Edit and fill in the attributes as applicable.
-Click **Save config**.
+### Configure your robot with an esp32
 
-The following attributes are available for `esp32` boards:
+The micro-RDK provides different component models than the fully featured RDK.
+See [Micro-RDK](/micro-rdk/) to get a list of supported models and instructions on how to configure them.
 
-<!-- prettier-ignore -->
-| Name | Type | Inclusion | Description |
-| ---- | ---- | --------- | ----------- |
-| `pins` | object | Required | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of any GPIO pins you wish to use as input/output with the [`GPIOPin` API](/program/apis/#gpio-pins). |
-| `analogs` | object | Optional | Attributes of any pins that can be used as analog-to-digital converter (ADC) inputs. See [configuration info](/components/board/#analogs). |
+Follow [this guide](/micro-rdk/board/esp32/) to configure an `esp32` board on your robot.
```

Comment:
I think this will make it stand out a little more
```suggestion
## Next Steps

{{< cards >}}
{{% manualcard link="/services/ml/" %}}

<h4>Configure your board </h4>

Sonfigure your `esp32` board for your robot.

{{% /manualcard %}}
{{< /cards >}}
```

Reply:
Awesome, thank you! Done